connector/src/main/java/com/google/cloud/spark/bigquery/SchemaConverters.java

```diff
@@ -15,19 +15,12 @@
  */
 package com.google.cloud.spark.bigquery;
 
-import com.google.cloud.bigquery.Field;
-import com.google.cloud.bigquery.FieldList;
-import com.google.cloud.bigquery.LegacySQLTypeName;
-import com.google.cloud.bigquery.Schema;
-import com.google.cloud.bigquery.StandardTableDefinition;
-import com.google.cloud.bigquery.TableDefinition;
-import com.google.cloud.bigquery.TableInfo;
-import com.google.cloud.bigquery.TimePartitioning;
+import com.google.cloud.bigquery.*;
```

Comment:
Please avoid import `package.*`, import the actual classes

Reply:
Done